src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECKeyXmlFormat.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// XML serialization formats for elliptic curve keys
+    /// </summary>
+    public enum ECKeyXmlFormat
+    {
+        /// <summary>
+        /// Use the format described in RFC 4050
+        /// </summary>
+        Rfc4050
```

Comment:
Style: trailing comma, please.  (Not that I hope we ever add another thing to this enum)